pkgs/os-specific/linux/mxu11x0/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub, kernel }:
+
+assert stdenv.isLinux;
+# it doesn't compile anymore on 3.14
+assert stdenv.lib.versionAtLeast kernel.version "3.18";
+
+let
+  rev = "de54053d6f297785d77aba9e9c880001519ffddf";
```

Comment:
I think you should either inline `rev` into the `fetchFromGitHub` call or lift the `sha256` up to here (whatever, so long as they are kept close).